app/uk/gov/hmrc/apiplatformoutboundsoap/services/OutboundService.scala

```diff
@@ -56,14 +61,33 @@ class OutboundService @Inject()(outboundConnector: OutboundConnector,
     outboundConnector.postMessage(envelope) flatMap { result =>
       val messageId = message.addressing.flatMap(_.messageId)
       val outboundSoapMessage = if (result < MULTIPLE_CHOICES) {
```

Comment:
HttpVerbs has the trait HttpErrorFunctions which has a handy is2xx method we could use

Reply:
Done